### PR TITLE
Generated doc fixes 4.5

### PR DESCRIPTION
**Pull request by fekete-robert**

### CODE REVIEW

File: pkg/sdk/logging/model/filter/concat.go [hunk 1]
```diff
@@ -79,8 +79,8 @@ type Concat struct {
 	PartialCriStreamKey string `json:"partial_cri_stream_key,omitempty"`
 }
 
-// ## Example `Concat` filter configurations
 /*
+## Example `Concat` filter configurations
```
Comment (pepov) [inline pkg/sdk/logging/model/filter/concat.go:84]: @fekete-robert this breaks go fmt, this is why the PR is failing

We could add an empty line comment if that works

Comment (fekete-robert) [reply pkg/sdk/logging/model/filter/concat.go:84]: Oh :( Then it'll probably break at some other places as well, I thought this pattern will work

Comment (pepov) [reply pkg/sdk/logging/model/filter/concat.go:84]: this breaks everywhere, just picked this as an example

Comment (fekete-robert) [reply pkg/sdk/logging/model/filter/concat.go:84]: Ok, let's discuss IRL which pattern should we use, then I'll fix it everywhere

**Review by pepov — APPROVED** [review]

**Pull request merged**